packages/commonwealth/client/styles/pages/manage_community/manage_roles.scss

```diff
@@ -21,6 +21,8 @@
       height: 24px;
       justify-content: space-between;
       width: auto;
+      padding-left: 4px;
+      padding-top: 4px;
```

Comment:
these will effect existing community management page. Lets make sure that new pages don't effect old UI.